web/server/entrypoint.go

```diff
@@ -54,6 +54,7 @@ type Arguments struct {
 	EnableFTDC                 bool   `flag:"ftdc,default=true,usage=enable fulltime data capture for diagnostics"`
 	OutputLogFile              string `flag:"log-file,usage=write logs to a file with log rotation"`
 	NoTLS                      bool   `flag:"no-tls,usage=starts an insecure http server without TLS certificates even if one exists"`
+	NetworkCheckOnly           bool   `flag:"network-check,usage=only runs normal network checks, logs results, and exits"`
```

Comment:
Try `go run ./web/cmd/server/main.go --network-check` on this branch and let me know what kind of output you get.